app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AdminsController < ApplicationController
+  def index
+    admin = Admin.all
+    render json: admin, status: :ok
+  end
+
+  def create
+    admin = Admin.create!(admin_params)
+    render json: admin, status: :created
+  end
+
+  def update
+    admin = Admin.update(admin_params)
+    render json: admin, status: :ok
+  end
+
+  def show
```

Comment:
Create a private method for finding a admin 
Create a private method for params permit

Reply:
done